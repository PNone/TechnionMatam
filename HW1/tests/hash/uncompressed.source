baraa regev 30 20:40
baraa regev 20 20:40
baraa regev 20 20:40
baraa regev 30 20:40
regev adi 15 20:30
baraa regev 30 20:25
daniel baraa 100 20:20
baraa daniel 20 20:20