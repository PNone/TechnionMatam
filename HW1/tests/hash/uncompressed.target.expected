07214b658f243aad98fd
ad8be1cf258e90073257
ad8be1cf258e90073257
07214b658f243aad98fd
16305a749e47698badcf
07214b658f243aad98fd
07ed4b218fcfe1332544
ad8be1cf25edc3210765